File: 30 Days of code/C/Day 27/soln.c
#include <stdio.h>

int main() {
    printf("5\n");

    /* YES */
    printf("5 3\n");
    printf("-1 80 888 10 0\n");

    /* NO */
    printf("4 2\n");
    printf("-1 0 1 2\n");

    /* YES */
    printf("3 3\n");
    printf("-1 0 1\n");

    /* NO */
    printf("6 1\n");
    printf("-1 0 1 2 3 4\n");

    /* YES */
    printf("7 3\n");
    printf("-1 0 1 2 3 4\n");

    return 0;
}

File: Java/Strings/Java String Reverse/soln.java
import java.io.*;
import java.util.*;

public class Solution {

    public static void main(String[] args) {
        Scanner scan = new Scanner(System.in);
        String str = scan.nextLine();
        scan.close();
        
        /* Reverse string and compare to original */
        String reversed = new StringBuilder(str).reverse().toString();
        System.out.println(str.equals(reversed) ? "Yes" : "No");
    }
}


